torch/csrc/serialization.cpp

```diff
@@ -110,11 +111,11 @@ static inline ssize_t doPythonIO(PyObject* fildes, void* buf, size_t nbytes, boo
 }
 
 // Call Python fildes.readinto(buf)
-ssize_t doPythonReadInto(PyObject* fildes, void* buf, size_t nbytes) {
+static inline ssize_t doPythonReadInto(PyObject* fildes, void* buf, size_t nbytes) {
   return doPythonIO(fildes, buf, nbytes, /* is_read */ true);
 }
 
 // Call Python fildes.write(buf)
-ssize_t doPythonWrite(PyObject* fildes, void* buf, size_t nbytes) {
+static inline ssize_t doPythonWrite(PyObject* fildes, void* buf, size_t nbytes) {
```

Comment:
Can you remove `inline` from those functions?

Reply:
Do you mean from just `doPythonReadInto`/`doPythonWrite` or also `isUnsupportedOperation` and `doPythonIO`? I don't have a good intuition for when to (or not to) inline a function.